concourse/replicator.py

```diff
@@ -524,10 +525,9 @@ def _notify_broken_definition_owners(self, failed_descriptor):
             branch=main_repo['branch'],
         )
 
-        codeowners_enumerator = CodeownersEnumerator()
         codeowners_resolver = CodeOwnerEntryResolver(github_api=github_api)
```

Comment:
for the sake of consistent API, I think COER should also be turned into functions, wdyt?

Reply:
will do